python/cudf/cudf/core/reshape.py

```diff
@@ -485,14 +486,13 @@ def melt(
     1  b         B          3
     2  c         B          5
     """
-    assert col_level in (None,)
+    assert col_level is None
```

Comment:
```suggestion
    if col_level is not None:
        raise NotImplementedError("col_level != None is not supported yet.")
```

We could change this assertion to a clear error message.